packages/contracts/src/contracts/PublicLock/PublicLockV14.sol

```diff
@@ -5347,7 +5347,7 @@ pragma solidity ^0.8.21;
  * Every ERC-721 compliant contract must implement the ERC165 interface.
  * https://eips.ethereum.org/EIPS/eip-721
  */
-contract PublicLockV14 is
+contract PublicLock is
```

Comment:
this is actually needed for consistency across version. All contracts are called `Unlock` or `PublicLock`, only the name of the file differs. That is the pattern that is used across the scripts. I changed it before but better to not number the contract name actually

Reply:
I actually moved this to its own PR as more changes are required (e.g. re packaging the bytecode) https://github.com/unlock-protocol/unlock/pull/13197